cmd/booster-http/gateway_handler.go

```diff
@@ -22,9 +29,11 @@ func newGatewayHandler(gw *BlocksGateway, supportedFormats []string) http.Handle
 		fmtsMap[f] = struct{}{}
 	}
 
+	// TODO: For the integration demo, we need to allow CORS requests to the gateway.
 	return &gatewayHandler{
-		gwh:              gateway.NewHandler(gateway.Config{Headers: headers}, gw),
+		gwh:              &corsHandler{gateway.NewHandler(gateway.Config{Headers: headers}, gw)},
```

Comment:
Definitely out of scope: I would prefer a variable name like `h` over `gwh`. Seems confusing to have multiplte "gatewayhandler" types nested like this.